definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -659,10 +655,9 @@ declare module "lodash" {
       object: T,
       iteratee?: ?ValueOnlyIteratee<A>
     ): { [key: V]: A, ... };
-    map<T, U>(array?: ?Array<T>, iteratee?: ?MapIterator<T, U>): Array<U>;
     map<T, U>(
-      array: ?$ReadOnlyArray<T>,
-      iteratee?: ReadOnlyMapIterator<T, U>
+      array?: ?$ReadOnlyArray<T>,
+      iteratee?: ?ReadOnlyMapIterator<T, U>
```

Comment:
`map<T, U>` above seems to be a duplicate which can be merged, since $ReadOnlyArray is a subtype of Array.